clang-tools-extra/test/clang-tidy/checkers/performance/noexcept-move-constructor-crash.cpp

```diff
@@ -0,0 +1,8 @@
+// RUN: %check_clang_tidy -std=c++17 -expect-clang-tidy-error  %s performance-noexcept-move-constructor %t 
```

Comment:
Does this need to be a separate file? Can't we just have it in an existing test, inside `namespace PRXXXXX`?